### PR TITLE
Minor contribution improvements to streamline on-boarding

### PR DESCRIPTION
**Pull request by HunterGerlach**

### CODE REVIEW

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -4,6 +4,9 @@
 name: Node.js Integration Tests
 
 on:
+  push:
```
Comment (hwchase17) [inline .github/workflows/integration.yml:7]: we dont quite have this set up with env vars yet

**Review by hwchase17 — COMMENTED** [review]
largely lgtm modulo the integration test stuff

**Comment by HunterGerlach** [comment]
Removed integration update+ref

File: package.json [hunk 2]
```diff
@@ -21,9 +21,12 @@
     "lint": "turbo run lint",
     "lint:fix": "yarn lint -- --fix",
     "test": "turbo run test",
+    "test:unit": "turbo run test --filter langchain --",
```
Comment (nfcampos) [inline package.json:24]: This is not quite true, we do have commands to run unit tests and integration tests, but they are inside the `langchain` folder

Comment (nfcampos) [reply package.json:24]: Specifically it's the next line that isn't correct, it can instead be `yarn workspace langchain run test:integration`

Comment (HunterGerlach) [reply package.json:24]: Yes, I actually went down the path of breaking out linting, unit tests, and integration tests into separate steps, but ultimately didn't think it provided much additional value when I noticed that you had the main parts covered in the CI action.

Comment (HunterGerlach) [reply package.json:24]: Oh, sorry I re-read that and I think I misunderstood what you were saying. I'll try to clean that up tomorrow. My goal was to have everything run from the root dir by default.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -76,25 +76,71 @@ Now, you should be able to run the common tasks in the following section.
...
+
 ### Testing
 
 Tests should be added within a `tests/` folder alongside the modules they
 are testing.
 
+To run all tests, run:
+
+```bash
+yarn test
```
Comment (nfcampos) [inline CONTRIBUTING.md:113]: This command would run only unit tests

**Pull request merged**